ranger-zookeeper/src/test/java/com/flipkart/ranger/zookeeper/model/SimpleServiceProviderTest.java

```diff
@@ -58,23 +58,36 @@ public void stopTestCluster() throws Exception {
         }
     }
 
+    private static class UnshardedInfo {
```

Comment:
USe `@Value`

Reply:
Not required in this particular case. But as a general idea, done in others!